pkg/asset/templates/kube-controller-manager.yaml

```diff
@@ -5,13 +5,11 @@ metadata:
   namespace: kube-system
   labels:
     k8s-app: kube-controller-manager
-    version: v1.3.0-beta.2
```

Comment:
This might just be down to being explicit with selectors https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/cluster-monitoring/standalone/heapster-controller.yaml#L27